portfolio/src/linter-checks

```diff
@@ -6,4 +6,6 @@ echo -e "\n######################################\n\n RUN HTML LINTERS \n"
 find . -type f -name \*.html -exec bash -c 'FN="{}"; npx htmllint "{}"' \;
 #
 echo -e "\n######################################\n\n RUN CSS LINTERS \n"
-find . -type f -name \*.css -exec bash -c 'FN="{}"; npx stylelint "{}"' \;
\ No newline at end of file
+find . -type f -name \*.css -exec bash -c 'FN="{}"; npx stylelint "{}"' \;
+echo -e "\n######################################\n\n RUN JAVA LINTERS \n"
+find . -type f -name \*.java -exec bash -c 'FN="{}"; /usr/lib/jvm/java-1.11.0-openjdk-amd64/bin/java -jar ../../google-java-format-1.8-all-deps.jar "{}" | diff --normal "{}"' \;
```

Comment:
nit: missing newline at the end of the file (for linter-checks and linter-fixes)